nautobot/extras/migrations/0024_job_data_migration.py

```diff
@@ -1,17 +1,33 @@
 from django.db import migrations
 
 from nautobot.core.fields import slugify_dots_to_dashes
+from nautobot.extras.constants import (
+    JOB_MAX_GROUPING_LENGTH,
+    JOB_MAX_NAME_LENGTH,
+    JOB_MAX_SLUG_LENGTH,
+)
 
 
 def _create_job_model(job_model_class, class_path):
     try:
         source, module_name, job_class_name = class_path.split("/")
+        # At this point in the migrations, source = "git.RepositoryName" not just "git", so not JOB_MAX_SOURCE_LENGTH
+        if len(source) > 110:
```

Comment:
How do we get to `110` and why isn't this a constant?

Reply:
This is the max_length value that's set in migration 0023. I can add a constant if you'd prefer but it's not a long-term value, it's changed to JOB_MAX_SOURCE_LENGTH in migration 0028.